app/controllers/submit.js

```diff
@@ -899,11 +1023,26 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
             const node = this.get('node');
             this.set('savingPreprint', true);
             this.toggleProperty('shareButtonDisabled');
-            model.set('isPublished', true);
-            node.set('public', true);
+
+            let log = null;
+            if (this.get('moderationType')) {
+                log = this.get('store').createRecord('reviewLog', {
+                   action: 'submit',
+                   reviewable: this.get('model')
+                });
+            }
+            if (this.get('moderationType') !== PRE_MODERATION) {
```

Comment:
Hmm, it should always set the node public on submission. The reviews stuff doesn't touch `node.is_public`, which I *think* is the correct way to be.